src/elasticsearch/donorIndexMapping.json

```diff
@@ -51,6 +51,12 @@
       "registeredTumourSamples": {
         "type": "byte"
       },
+      "RNApublishedNormalAnalysis": {
```

Comment:
Adding two new fields for RNA raw reads columns(new feature for next sprint), not really in the scope of this ticket, but was a quick implementation and kinda related to this bug.

Reply:
Camelcase naming is weird here:
`rnaPublishedNormalAnalysis` and `rnaPublishedTumourAnalysis` looks better to my eyes

Reply:
Would update this throughout.